cmd/util/flags/cliflags/spec.go

```diff
@@ -35,7 +35,7 @@ func NewSpecFlagDefaultSettings() *SpecFlagSettings {
 type SpecFlagSettings struct {
 	Publisher     opts.PublisherOpt // Publisher - publisher.Publisher
 	Inputs        opts.StorageOpt   // Array of inputs
-	OutputVolumes []string          // Array of output volumes in 'name:mount point' form
+	OutputVolumes map[string]string // Map of output volumes in 'name:mount point' form
```

Comment:
This maps to a slice of `models.ResultPath`. Using a map felt more intuitive and idiomatic, as it allows for direct enforcement of the name-to-mount mapping through the flags library, rather than performing the validation after the fact.